colorschemes/gruvbox-dark.conf

```
@@ -0,0 +1,164 @@
+#
+# Copyright (C) 2025 Lucas Vieites <lucas.vieites@gmail.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+# MA 02110-1301, USA.
+#
+# Ported by Lucas Vieites <lucas(dot).vieites(at)gmail(dot)com>
+# Color scheme by morhetz: https://github.com/morhetz/gruvbox-contrib/blob/master/gedit/gruvbox-dark.xml
+#
+
+[theme_info]
+name=Gruvbox Dark
+description=A dark color scheme using the Gruvbox palette
+# incremented automatically, do not change manually
+version=0.1
+author=Lucas Vieites <lucas.vieites@gmail.com>
+url=https://github.com/lucasvieites/toolbox/upload/master/config_files/gruvbox-dark.conf
```

The URL seems to be to upload a file.
```suggestion
url=https://github.com/lucasvieites/toolbox/blob/master/config_files/gruvbox-dark.conf
```